Devscord.DiscordFramework/Services/DirectMessagesService.cs

```diff
@@ -52,6 +53,17 @@ public async Task<bool> TrySendMessage(ulong userId, string message, MessageType
         public async Task<bool> TrySendEmbedMessage(ulong userId, string title, string description, IEnumerable<KeyValuePair<string, string>> values)
         {
             var embed = this._embedMessagesService.Generate(title, description, values);
+            return await TrySendEmbedMessage(userId, embed);
+        }
+
+        public async Task<bool> TrySendEmbedMessage(ulong userId, string title, string description, Dictionary<string, Dictionary<string, string>> values)
+        {
+            var embed = this._embedMessagesService.Generate(title, description, values);
+            return await TrySendEmbedMessage(userId, embed);
+        }
```

Comment:
```suggestion
        public Task<bool> TrySendEmbedMessage(ulong userId, string title, string description, Dictionary<string, Dictionary<string, string>> values)
        {
            var embed = this._embedMessagesService.Generate(title, description, values);
            return TrySendEmbedMessage(userId, embed);
        }
```

Reply:
każdy await to stworzony kolejny wątek w tle https://stackoverflow.com/a/19985988

Reply:
a jaki jest z tym problem? also wydaje mi się, że skoro mamy tutaj wysyłanie rzeczy do internetu, czyli potencjalnie długą operację, to powinna być asynchroniczna - ale popraw mnie, jeśli źle rozumuję

Reply:
ona jest asynchroniczna, na ten task który zwracasz ktoś kto go wywoła, będzie musiał zaczekać dając await

Reply:
teraz jest zagnieżdzony await po prostu

Reply:
chciała* ;)
dzięki, obczaję